src/index.html

```diff
@@ -1,12 +1,395 @@
+    <meta name="viewport"
+          content="
+            width=device-width,
+            initial-scale=1.0"
+    >
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#home">
+        <img
+          class="header__logo"
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link nav__link--is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link"
+            data-qa="nav-hover" >Laptops & computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <h1 class="visually-hidden">Product catalog</h1>
+
+      <section class="cards">
+        <article class="card cards__card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac27inches"
+          >
+          <h2 class="card__tittle">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <div class="card__code">
+            Product code: 195434
+          </div>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+              <span class="stars__star"></span>
+            </div>
+            <span class="card__review-text">
+              Reviews: 5
+            </span>
+          </div>
+          <div class="card__price-container">
+            <div class="card__price-tag">
+              Price:
+            </div>
+            <div class="card__price-amount">
+              $2,199
+            </div>
+          </div>
```

Comment:
```suggestion
          <p class="card__price-container">
            <span class="card__price-tag">
              Price:
            </span>
            <span class="card__price-amount">
              $2,199
            </span>
          </p>
```
Not a mistake, but instead of nesting divs since it's just some text you can wrap it inside a p tag and use span for the text. It will be easier to tell what's the content of this block.